src/ocaml/utils/load_path.ml

```diff
@@ -31,17 +31,8 @@ module Dir = struct
   let path t = t.path
   let files t = t.files
 
-  (* For backward compatibility reason, simulate the behavior of
-     [Misc.find_in_path]: silently ignore directories that don't exist
-     + treat [""] as the current directory. *)
-  let readdir_compat dir =
-    try
-      Sys.readdir (if dir = "" then Filename.current_dir_name else dir)
```

Comment:
Shouldn't we keep this function and just replace this `Sys.readdir` by `Directory_content_cache.read`?

Reply:
This function is already  in Directory_content_cache: https://github.com/ocaml/merlin/blob/b6ac38a5b1ce9b65e1afdaa35d83c2e4f323b8a3/src/ocaml/utils/directory_content_cache.ml#L5